src/js/lib/client/src/index.js

```diff
@@ -177,6 +176,15 @@ function parseCmdArgs() {
 
 let args = parseCmdArgs()
 
+// Note: this is a conditional default to avoid issues with some window managers affecting
+// interactions at the top of a borderless window. Thus, we want borders on Win/Linux and
+// borderless on Mac. See https://github.com/enso-org/ide/issues/1101 for details.
```

Comment:
Could you also add issue on Electron bug tracker for this? Currently, the `-webkit-app-region  : drag;` prevents events to be send to JS on Windows and Linux, but works on MacOS. This is used to define custom dragregion, as described here https://www.electronjs.org/docs/api/frameless-window#draggable-region